pennylane/transforms/qcut.py

```diff
@@ -524,8 +524,12 @@ def expand_fragment_tapes(
                         apply(op)
 
                 with qml.tape.stop_recording():
-                    measurements = _get_measurements(group, tape.measurements)
-
+                    if len(group) > 0:
+                        measurements = _get_measurements(group, tape.measurements)
+                    # This ensures the measurements of the original tape are carried over to the
+                    # following tape configurations in the absence of any MeasureNodes in the fragment
+                    else:
+                        measurements = tape.measurements
```

Comment:
We could put this logic in `_get_measurements`, i.e., the first thing we do is:
```
if len(group) == 0:
    return measurements
```

Reply:
Nice idea! I've moved it in this commit 1129df69de8503e3b62f918cdedc5afed7237ec1